doc/source/conf.py

```diff
@@ -309,3 +310,34 @@ def download_and_process_files(example_links: List[str]) -> List[str]:
 jinja_globals = {
     "ANSYS_SPHINX_THEME_VERSION": version,
 }
+
+
+def revert_exclude_patterns(app, env):
+    """Revert the exclude patterns.
+
+    Parameters
+    ----------
+    app : Sphinx
+        Sphinx application instance.
+    env : BuildEnvironment
+        The build environment.
+
+    Notes
+    -----
+    Remove the examples/gallery-examples/*.ipynb pattern from the exclude patterns.
+    When the nbsphinx extension is enabled, the exclude patterns are modified
+    to exclude the examples/gallery-examples/*.ipynb pattern. This function reverts
+    the exclude patterns to their original state.
+    """
+    excluded_pattern = env.config.exclude_patterns
+    # HACK: Remove the examples/gallery-examples/*.ipynb pattern from the exclude patterns
+    # for nbshpinx to build the gallery examples
+    # If examples/gallery-examples/*.ipynb is not removed, the nbsphinx will
+    # try to build the gallery examples and fail.
```

Comment:
Let's document this only in one place: the docstring description, its notes, or as a comment.